pkg/operator/defaultscccontroller/defaultscc.go

```diff
@@ -0,0 +1,103 @@
+package defaultscccontroller
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+
+	"github.com/openshift/api"
+	securityv1 "github.com/openshift/api/security/v1"
+
+	assets "github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/defaultscc_assets"
+)
+
+// DefaultSCCCache maintains the list of default SecurityContextConstraints objects
+// shipped wth an OpenShift cluster.
+type DefaultSCCCache struct {
+	// When we iterate through the default SCC we want to maintain a distinct order.
+	names []string
+	set   map[string]*securityv1.SecurityContextConstraints
+}
+
+func (d *DefaultSCCCache) IsDefaultSCC(name string) bool {
```

Comment:
merge this and the next func in the more common `Get(string) (*SCC, bool)` interface.

Reply:
Do we need this func anymore?

Reply:
removed it